micronaut/src/test/kotlin/test/micronaut/deprecation/none/NoDeprecation.kt

```diff
@@ -0,0 +1,11 @@
+package test.micronaut.deprecation.none
+
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+
+@Controller("/todos", produces = ["text/plain", "application/xml"])
```

Comment:
Since you have a Unit return type, you can omit the produces. Otherwise it might just be confusing ;)